.github/workflows/continuous-integration.yml

```
@@ -33,7 +33,10 @@ jobs:
             composer-options: "--prefer-dist --ignore-platform-req=php+"
           - php-version: "8.5"
             composer-options: "--prefer-dist --ignore-platform-req=php+"
-            
+          - php-version: "8.4"
+            composer-options: "--prefer-dist --ignore-platform-req=php+"
+            composer-minimum-stability: "dev"
+            symfony-version: "8.0.x-dev"
```

We already have beta tags: [v8.0.0-BETA1](https://packagist.org/packages/symfony/console#v8.0.0-BETA1)

```suggestion
            symfony-version: "^8.0.0-BETA1"
```